HW01/Vadim_Volovenko/vadim_volovenko_hw_01_t_01.rb

```diff
@@ -0,0 +1,10 @@
+def calculate_angle(hours, minutes)
+  # one hour = 360/12 = 30 degrees
+  hours_angle = hours * 30 + minutes * 0.5
+  # one minute = 360/60 = 6 degrees
+  minutes_angle = 6 * minutes
```

Comment:
we usually use a different order
```suggestion
  minutes_angle = minutes * 6
```